.github/workflows/llm-nightly-test.yml

```diff
@@ -14,7 +14,7 @@ on:
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   llm-nightly-convert-test:
-    runs-on: [ self-hosted, Gondolin, ubuntu-20.04-lts ]
+    runs-on: [ llm, AVX512, ubuntu-20.04-lts ]
```

Comment:
Maybe `self-hosted` label need to be added.
[ref](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/using-self-hosted-runners-in-a-workflow)
other LGTM